### PR TITLE
test: set longer timeout

### LINKED ISSUE #12268: TEST-30-ONCLOCKCHANGE is very flakey
It failed on maybe half of PRs recently, e.g.
https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-bionic-upstream-systemd-ci-systemd-ci/bionic/amd64/s/systemd-upstream/20190410_134709_9d4d9@/log.gz

### PR DESCRIPTION
**Pull request by yuwata**
Not sure, but hopefully fixes #12268.

### CODE REVIEW

File: test/TEST-30-ONCLOCKCHANGE/test.sh [hunk 1]
```diff
@@ -29,6 +29,12 @@ test_setup() {
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-resolved.service
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-machined.service
 
+        # extend the watchdog
+        cat >$initdir/etc/systemd/system/testsuite.service <<EOF
+[Service]
+WatchdogSec=10min
+EOF
```
Comment (poettering) [inline test/TEST-30-ONCLOCKCHANGE/test.sh:36]: hmm, wah? we write this service file again a few lines further down?

Comment (yuwata) [reply test/TEST-30-ONCLOCKCHANGE/test.sh:36]: ouch. fixed...

File: test/TEST-30-ONCLOCKCHANGE/test.sh [hunk 2]
```diff
@@ -29,6 +29,12 @@ test_setup() {
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-resolved.service
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-machined.service
 
+        # extend the watchdog
+        cat >$initdir/etc/systemd/system/systemd-timedated.service.d/watchdog.conf <<EOF
```
Comment (keszybz) [inline test/TEST-30-ONCLOCKCHANGE/test.sh:33]: Does the directory exist already? I guess it must since the test passes ;_)

Comment (yuwata) [reply test/TEST-30-ONCLOCKCHANGE/test.sh:33]: ouch...

Comment (yuwata) [reply test/TEST-30-ONCLOCKCHANGE/test.sh:33]: #12278.

**Pull request merged**